services/streammanager/kafka/kafkamanager.go

```diff
@@ -9,18 +9,23 @@ import (
 	"strings"
 	"time"
 
-	"github.com/tidwall/gjson"
-
+	"github.com/linkedin/goavro"
 	"github.com/rudderlabs/rudder-server/config"
 	"github.com/rudderlabs/rudder-server/services/stats"
 	"github.com/rudderlabs/rudder-server/services/streammanager/kafka/client"
 	rslogger "github.com/rudderlabs/rudder-server/utils/logger"
+	"github.com/tidwall/gjson"
 )
 
 type Opts struct {
 	Timeout time.Duration
 }
 
+// schema is the AVRO schema required to convert the data to AVRO
+type schema struct {
+	avroSchema string
+}
```

Comment:
why do we need this struct ?
can't we do something like:
`type avroSchema string`
plus the name `schema` is a little bit missleading

Reply:
We could do it without the struct I think. About the name, I think it is very common to refer to them as schemas.